examples/js/loaders/GLTFLoader.js

```diff
@@ -371,7 +371,8 @@ THREE.GLTFLoader = ( function () {
 
 		lightNode.name = lightDef.name || ( 'light_' + lightIndex );
 
-		return Promise.resolve( lightNode );
+		return Promise.
+		( lightNode );
```

Comment:
typo here?

Reply:
github's find/replace UI is weird this has happened to me before 😅

Reply:
strange as always  :-)